.github/workflows/rustfmt.yml

```diff
@@ -0,0 +1,24 @@
+name: rustfmt
+on: [push, pull_request]
+jobs:
+
+  rustfmt:
+    name: Format
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install nightly toolchain with rustfmt available
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
```

Comment:
Does tendril require nightly to build? If not, then it should just require stable instead of nightly.

Reply:
No, it doesn't. I'll fix it.